New_PlatformGame/Assets/NewPlayerMovement.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+public class NewPlayerMovement : MonoBehaviour
+{
+    //public 변수
+    public float jumpForceMax = 200f; //점프하는 힘의 최대치
+    public Sprite gotJumpBarSprite; //점프바의 스프라이트를 받기위한 변수
+    public float jumpChargeSpeed = 1f; //점프하는 힘을 더하는 속도
+    public float jumpBarLength; //점프바의 길이
+    public float xMoveSpeed = 5f; //움직임 속도
+    public float xMoveSpeedFly = 1f; //공중에 떠있을때 움직임 속도
+    public float jumpBarPositionX = 1f; //점프바의 위치 X (캐릭터 기준)
+    public float jumpBarPositionY = 1f; //점프바의 위치 Y (캐릭터 기준)
+
+    //private 변수
+    private Rigidbody2D _rigidbody2D;
+    private Collider2D _collider2D;
+    private GameObject _jumpBar; //점프바 변수
+    private SpriteRenderer _jumpBarSprite; //점프바의 스프라이트 렌더러
+    private float _xMove;
+    private Vector2 _getvel; //Move 함수에서 캐릭터의 속도를 받는 변수
```

Comment:
```suggestion
    private Vector2 _getvel; //Move 함수에서 캐릭터의 속도를 받는 변수
```
`카멜 표기법 잘 지키기` 규칙이 지켜지지 않았습니다. `getvel` -> `getVel` 로 고쳐야하고, Vel을 Velocity로 풀어 쓰는 것을 추천합니다.